Workflow/Sources/WorkflowHost.swift

```diff
@@ -80,11 +78,69 @@ public final class WorkflowHost<WorkflowType: Workflow> {
         handle(output: output)
     }
 
+    /// Add a listener closure to be called back when a render pass happens.
+    public func addRenderingListener(_ closure: @escaping (WorkflowType.Rendering) -> Void) -> UUID {
```

Comment:
It actually broke tests behavior to do that.  Before people could subscribe to a signal created from the rendering property which would only fire when rendering changed.  If we call it here it will fire immediately.
They still have the option of taking the current rendering from the property on WorkflowHost and call the listener with it themselves to trigger it.